src/grouping.js

```diff
@@ -0,0 +1,17 @@
+const { getCardNumber } = require("./getCardNumber");
+
+const numbers = hand.map(getCardNumber);
```

Comment:
You are doing work outside of the function meaning that when it is called it won't work as expected because the part outside the function won't run.